spark_api.gemspec

```diff
@@ -28,33 +28,26 @@ Gem::Specification.new do |s|
   s.files              = Dir["{History.txt,LICENSE,Rakefile,README.md,VERSION}", "{bin,lib,script}/**/*"]
   s.test_files         = Dir["spec/{fixtures,unit}/**/*", "spec/*.rb"]
   s.executables        = ["spark_api"]
-  s.default_executable = %q{spark_api}
   s.require_paths      = ["lib"]
   
-  s.add_dependency 'faraday', '~> 0.9.0'
+  s.add_dependency 'faraday', '>= 0.16.0'
   s.add_dependency 'multi_json', '~> 1.0'
-  s.add_dependency 'json', '~> 1.7'
+  s.add_dependency 'json', '>= 1.7'
   s.add_dependency 'builder', '>= 2.1.2', '< 4.0.0'
   s.add_dependency 'will_paginate', '>= 3.0.pre2', '< 4.0.0'
   s.add_dependency 'highline', '>= 1.0'
 
-  # spark_api doesn't use public_suffix directly. spark_api uses Webmock, which 
-  # uses addressable, which uses public_suffix. Bundler has suddenly started 
-  # trying to install public_suffix 2.0.4, which requires Ruby 2. When spark_api 
-  # starts to require Ruby 2, this dependency can be removed.
-  s.add_development_dependency 'public_suffix', '~> 1.4.6'
-
   # TEST GEMS
-  s.add_development_dependency 'rake', '~> 0.9.2'  
-  s.add_development_dependency 'rspec', '~> 2.14.0'
-  s.add_development_dependency 'webmock', '~> 1.9'
-  s.add_development_dependency 'typhoeus', '~> 0.3'
-  s.add_development_dependency 'ci_reporter', '~> 1.7.0'
+  s.add_development_dependency 'rake', '>= 13.0'
+  s.add_development_dependency 'rspec', '>= 3.0'
```

Comment:
Thisi isn't a showstopper, but could we clean up the deprecation warnings triggered by the rspec upgrade?

Reply:
I got a couple of them, but the :should syntax warnings will take refactoring some tests which I'd rather avoid for now

Reply:
Can we skip the Rspec upgrade instead? My concern is if we don't do that now it'll just get punted. If it's usable, the refactoring work was done in the v2 branch:
 - 7573c62ed462e2f218f3d04db0e27371f8356f59
 - 83fd57af46194adb7c267490a3b057a5e2674e54
 - 2fcae6d1db838fe4cf1e20524f793df71b157fef
 - b8ca5090377f9dfc62ac7a0b12c13c9fcc16e5ee
 - fc5d28823b84cf8072da559a01d6200a0b6f448a

Reply:
The gems were all updated to the minimal versions that work.  I agree that deprecation warnings aren't ideal, but if the tests are already refactored, can we pull those changes in a separate PR?